pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -213,7 +212,6 @@ def on_pretrain_routine_start(self, trainer, pl_module):
         When pretrain routine starts we build the ckpt dir on the fly
         """
         self.__resolve_ckpt_dir(trainer)
-        self.save_function = trainer.save_checkpoint
```

Comment:
couldn't find any references in our callbacks or bolts, but perhaps leave it for backward compatibility?